dask_ml/model_selection/_split.py

```diff
@@ -450,19 +452,54 @@ def train_test_split(
 
     if all(isinstance(arr, (dd.Series, dd.DataFrame)) for arr in arrays):
         check_matching_blocks(*arrays)
-        if blockwise is None:
-            blockwise = False
+        if blockwise is False:
+            raise NotImplementedError(
+                "'blockwise=False' is not currently supported for dask DataFrames."
+            )
 
         rng = check_random_state(random_state)
         rng = draw_seed(rng, 0, _I4MAX, dtype="uint")
-        return list(
-            itertools.chain.from_iterable(
-                arr.random_split([train_size, test_size], random_state=rng)
-                for arr in arrays
+        if DASK_GT_2130:
+            if shuffle is None:
+                shuffle = False
+                if not sys.warnoptions:
```

Comment:
What is this? Typically we would just warn, and the users can filter it using the usual filters.